theme/partials/header.html

```diff
@@ -69,22 +70,132 @@ <h2>Meet <br> Qlik Core<sup>&reg;</sup></h2>
                 and visualizations with this development platform <br>
                 built on the Qlik Associative Engine.         
             </div>
-                                         
-            
-          </div>   
-          <div class="md-flex__cell md-flex-header-buttons">
             <a href="licensing/">
-              <div class="md-header-button first-button">Try for Free</div>
+              <div class="md-header-button">Try for Free</div>
             </a>
             <a href="get-started/">
               <div class="md-header-button">Start Tutorials</div>
-            </a>
-          </div>    
-          <div class="md-flex__cell md-flex__cell--shrink">
+            </a>                             
             
+          </div>   
           </div>
       </div>
       {% endif %}
     </nav>
   </header>
+  {% if page.is_homepage %}
+  <style>
+    .md-content, .md-header-nav__title {
+      display: none !important;
+    }
+
+    .md-main__inner {
+      height: 1px;
+    }
+
+    .md-container {
+      height: initial;
+    }
+
+    @media (min-width: 76.1875em) {
+      .md-sidebar {
+        display: none !important;
+      }
+    }
+  </style>
+  <header class="header-intro">
+    <nav class="md-header-nav md-grid">
+      <div class="md-flex">
+        <div class="md-flex__cell md-flex__cell--stretch head-home">
+          <h1>HELLO</h1>
+          <p>With Qlik Core, you can take advantage of Qlik's powerful associative engine to build, 
+          extend and quickly deploy custom interactive data-driven solutions that are highly scalable and cloud-ready.</p>
+          <p>And built for Linux and running in a Docker container, Qlik Core applications can easily be embedded into any stack, 
```

Comment:
Adding `and` made it repetitive

Reply:
@TRB3269 comments?

Reply:
To me, this makes it flow a little better from the last sentence. These sentences should also be shown as one paragraph...

With Qlik Core, you can take advantage of Qlik's powerful associative engine to build, extend and quickly deploy custom interactive data-driven solutions that are highly scalable and cloud-ready. And built for Linux and running in a Docker container, Qlik Core applications can easily be embedded into any stack, IoT device, process, application, web page, portal, Star Wars visualization – you name it.

Reply:
Felt like it became too much text when putting it as one paragraph, but I'll change it and you can see if it feels better.